resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
+                "exp": 10,
+                "stat_trait": ["bloodthirsty","vengeful","arrogant","troublesome"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["dislike","respect"],
+                        "amount": 10
+                    },
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["trust","platonic","comfort"],
+                        "amount": -10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l's whiskers twitch in annoyance at this childish outburst. If app1 doesn't <i>want</i> to be a warrior, that's fine by p_l. {PRONOUN/p_l/subject}'ll be in the training grounds, waiting to train the greatest warrior c_n's ever seen. {PRONOUN/p_l/subject} {VERB/p_l/walk/walks} away from app1, and a hidden smile dashes across {PRONOUN/p_l/poss} muzzle as {PRONOUN/p_l/subject} {VERB/p_l/hear/hears} app1 start to chase after {PRONOUN/p_l/poss}.",
+                "exp": 10,
+                "stat_trait": ["cold","strict","grumpy","lonesome","gloomy"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l lashes {PRONOUN/p_l/poss} tail. Of course app1 will never be a warrior if {PRONOUN/app1/subject} {VERB/app1/think/thinks} like <i>that.</i> c_n doesn't want warriors who think like <i>that.</i> Half of glory, {PRONOUN/p_l/subject} {VERB/p_l/explain/explains} to app1, is believing you deserve it. P_l challenges app1: focus on <i>that</i>. When p_l and app1 return to the training grounds, the other trainees notice app1 fighting with a newfound vigor.",
+                "exp": 10,
+                "stat_trait": ["fierce","bold","confident","competitive","ambitious","rebellious","daring","adventurous","cunning"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l's heart lurches seeing the apprentice so low. Could p_l have done something to make app1 feel this way? Not having any words p_l sits close to app1, wrapping {PRONOUN/p_l/poss} tail around {PRONOUN/app1/poss}. P_l purrs softly, reassuringly. Until app1 purrs back, gratefully. ",
```

Comment:
awww so cute! 
also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l
can't capitalise p_l